Use isomorphic-ws

To make this compatible with the web browser, use isomorphic-ws instead of ws.

@achingbrain I can have a look at making a repro, but I'm running into this again-

importing it-ws imports "events" which isn't available on the browser side:

```
✘ [ERROR] Could not resolve "events"

    ../../../../node_modules/it-ws/dist/src/server.js:5:29:
      5 │ import { EventEmitter } from 'events';
        ╵                              ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "Platform: api.PlatformNode" to do that, which will remove this
  error.
```

--- a/package.json
+++ b/package.json
@@ -203,7 +203,7 @@
   },
   "browser": {
     "./dist/src/web-socket.js": "./dist/src/web-socket.browser.js",
-    "./server": false,
+    "./dist/src/server.js": false,
     "ws": false,
     "http": false,
     "https": false

